packages/salesforcedx-vscode-apex/src/index.ts

```diff
@@ -139,7 +139,7 @@ export async function activate(context: vscode.ExtensionContext) {
 }
 
 function registerCommands(
-  extensionContext: vscode.ExtensionContext
+  // extensionContext: vscode.ExtensionContext
```

Comment:
@jeffb-sfdc was this comment intentional?

Reply:
It's not used.  I'll go and remove it.